### ISSUE TITLE
Persistent 'Unable to get token balance' on Wallet with Ropsten (LES) network even sync finished

### ISSUE THREAD

**Issue by Serhy** [status-im/status-react#6904]
***Type***: Bug

***Summary***: ETH and token balance are always 0 on Wallet for the contact who use Ropsten (LES) network. And `Unable to get token balance` on Wallet main screen displays all the time, even the network sync finished (current block = latest LES block).
Despite the latest block number is not really the latest (see https://github.com/status-im/status-react/issues/6901 for more info) I checked with the account with transaction history that started from 3415709 block (while LES block is around `#4329035` and 'Not syncing' displays on Home screen). 

#### Expected behavior
Once LES is finished sync and current block = latest block, - ETH and token balance on Wallet main screen is the same as with Ropsten RPC 

#### Actual behavior
Once LES is finished sync and current block = latest block, - ETH and token balance on Wallet main screen are all 0. 'Unable to get token balance' error on Wallet main screen.

<img width="359" alt="screenshot 2018-11-27 at 15 39 10" src="https://user-images.githubusercontent.com/8749671/49085404-9b394400-f25a-11e8-8ade-b628d97b4f7e.png">

### Reproduction

- Open Status develop build (e.g. nightly) and create new account (or recover existing account with ETH and/or token)
- Switch fleet setting to `les.dev.ropsten` and set the network to `Ropsten (LES)`
- Stay on Home screen until 'Not syncing' is displayed on Home screen
- Navigate to Wallet

### Additional Information
* Status version: Develop `0.9.31 (1811211215); node 52a1bdfe`
* Operating System: Android and iOS

#### Logs
[logcat.txt](https://github.com/status-im/status-react/files/2620089/logcatunabletogettoken.txt) after app launched and sync is 100%: 
[gethLogLevelTrace.log](https://github.com/status-im/status-react/files/2620090/geth.11.log)

**Comment by mandrigin** [comment]
Does it show “Not syncing” when you request? Because even if the current block number is equal to the latest one, it doesn’t mean that the sync is finished, it is still ongoing.

**Comment by Serhy** [comment]
Yes, 'Not syncing' is what displayed on Home screen.  

So syncing started from block ~4161535 `...from 4329035 blocks`. Eventually sync ended (this I confirmed in geth.log's rows similar to ```t=2018-11-28T10:28:39+0000 lvl=dbug msg="Inserted new header" number=4329035 hash=0x0229353baf14b4c93c5b36eccae2bce9c5c606c7be805655424b8d80c316a821``` ) and on Home screen 'Not syncing' under LES info displayed (along with the block number = 4329035). 

Updated steps in description to wait until LES sync reach 'Not syncing' level on Home screen.
